cmake/modules/shields.cmake

```diff
@@ -79,41 +79,56 @@ foreach(root ${BOARD_ROOT})
 
       list(REMOVE_ITEM SHIELD-NOTFOUND ${s})
 
-      # Add <shield>.overlay to the shield_dts_files output variable.
-      list(APPEND
-        shield_dts_files
-        ${SHIELD_DIR_${s}}/${s}.overlay
-        )
+      # Add <shield>.overlay to a temporary variable
+      set(shield_${s}_dts_file ${SHIELD_DIR_${s}}/${s}.overlay)
 
-      # Add the shield's directory to the SHIELD_DIRS output variable.
-      list(APPEND
-        SHIELD_DIRS
-        ${SHIELD_DIR_${s}}
-        )
+      # Add the shield's directory to a temporary variable
+      set(shield_${s}_dir ${SHIELD_DIR_${s}})
```

Comment:
is this actually needed ?
`SHIELD_DIR_${s}` already seems to be shield specific.
```suggestion
```

Reply:
Fixed